src/routes/about.svelte

```diff
@@ -1,44 +1,57 @@
+<script>
+	import './about.scss';
+</script>
 <svelte:head>
-	<title>Frescomer | La mesa está servida</title>
+	<title>Feed Me Tj</title>
 </svelte:head>
-<h1>Frescomer | La mesa está servida</h1>
+<div class="about-page">
+	<div class="title">
+		<h1><img src="fmt-logo-gold.svg" alt="Feed Me Tj"></h1>
+	</div>
+	<div class="content-block secondary">
+		<h2>¿Qué es Feed Me Tj?</h2>
+		<p>FMT es un directorio que hace muy fácil publicar y ver ofertas para comer cerca de tu ubicación.</p>
+		<p>Nuestro contenido viene de Instagram a través de hashtags que estamos observando. Si agregas <span class="fancy">#feedmetj</span> a tu post, aparecerá en FMT.</p>
```

Comment:
queremos empezar la cura `#feedmetj` ?

los hashtags que seguimos son:
#tijuanamakesmehungry 
#tijuanafood

Reply:
No sugiero que los dejemos de seguir, pero si el proceso es seguir hashtags y tenemos que informar eso, pues let's tell them to use ours.